src/ddscxx/include/org/eclipse/cyclonedds/sub/qos/DataReaderQosDelegate.hpp

```diff
@@ -80,6 +81,7 @@ class OMG_DDS_API DataReaderQosDelegate
     dds::core::policy::Ownership               ownership_;
     dds::core::policy::TimeBasedFilter         tfilter_;
     dds::core::policy::ReaderDataLifecycle     lifecycle_;
+    dds::core::policy::DataRepresentation      datarepresentation_;
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro. (BTW: I see that in this particular case the embedding OMG_DDS_OWNERSHIP_SUPPORT macro was also missing for the "ownership_" attribute.)